submissions/ArthurGorbenko/HTML5-Form/index.html

```diff
@@ -0,0 +1,487 @@
+                id="form-upload"
+                class="visually__hidden"
+              />
+            </label>
+            <table class="form-upload_table">
+              <thead>
+                <th class="form-table_cell form-table_cell_headers">
+                  File name
+                </th>
+                <th class="form-table_cell form-table_cell_headers">
+                  File size
+                </th>
+                <th class="form-table_cell form-table_cell_headers">
+                  Number of words
+                </th>
+                <th class="form-table_cell form-table_cell_headers">remove</th>
+              </thead>
+
+              <tbody>
+                <tr class="form-upload_table_row">
+                  <td class="form-table_cell form-table_cell_file_name">
+                    somedocs.doc
+                  </td>
+                  <td class="form-table_cell">78Kb</td>
+                  <td class="form-table_cell">823</td>
+                  <td class="form-table_cell form-table_cell_remove"></td>
+                </tr>
+                <tr class="form-upload_table_row">
+                  <td class="form-table_cell form-table_cell_file_name">
+                    somedocs.doc
+                  </td>
+                  <td class="form-table_cell">78Kb</td>
+                  <td class="form-table_cell">823</td>
+                  <td class="form-table_cell form-table_cell_remove"></td>
+                </tr>
+              </tbody>
+            </table>
+          </section>
+          <section class="form-language_choice">
+            <span class="form-sections_headers_text form-language_choice_header"
+              >Choose your languages pairings</span
+            >
+            <span class="form-select_custom">
+              <span class="select_from"></span>
+              <select name="" id="" class="button form-selects" required>
+                <option value="eng">English</option>
+                <option value="german">German</option>
+                <option value="сhinese">Chinese</option>
+              </select>
+            </span>
+            <div class="wrapper_form-selects">
+              <span class="select_from select_to"></span>
+              <span class="form-select_custom form-selects_bottom">
+                <select name="" id="" class="button form-selects">
+                  <option value="arabian">Arabian</option>
+                  <option value="hindi">Hindi</option>
+                  <option value="french">French</option>
+                </select>
+              </span>
+              <span class="form-selects_bottom form-select_custom">
+                <select name="" id="" class="button form-selects">
+                  <option value="hindi">Hindi</option>
+                  <option value="arabian">Arabian</option>
+                  <option value="french">French</option>
+                </select>
+                <div class="form-select_remove"></div>
```

Comment:
This must be a button. Look on element's purpose not how he looks.

Reply:
Agree,done.